gui/src/test/java/kcl/teamIndexZero/traffic/gui/GUI_PrimitivesTest.java

```diff
@@ -0,0 +1,48 @@
+package kcl.teamIndexZero.traffic.gui;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by kumarawijeet on 3/16/2016.
+ */
+public class GUI_PrimitivesTest {
```

Comment:
Yeah, as I said in the chat, I don't see any thing that needs too be tested, as the picture is being produced and when we change values then too it's working without any problem, so basically there is no need for this. As I said earlier also, if you see anything that I should test for let me know, I am not too versed with testing files though so...                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the EE network.                                                                                                                                                                                                                From: Alexander CherednichenkoSent: Sunday 20 March 2016 23:20To: teamIndexZero/index_zero_trafficsystemReply To: teamIndexZero/index_zero_trafficsystemCc: kumar awijeetSubject: Re: [index_zero_trafficsystem] Gui graphics kumar awijeet (#9)In gui/src/test/java/kcl/teamIndexZero/traffic/gui/GUI_PrimitivesTest.java:
> @@ -0,0 +1,48 @@
> +package kcl.teamIndexZero.traffic.gui;
> +
> +import org.junit.After;
> +import org.junit.Before;
> +import org.junit.Test;
> +
> +import static org.junit.Assert.*;
> +
> +/**
> + * Created by kumarawijeet on 3/16/2016.
> + */
> +public class GUI_PrimitivesTest {

What's the reason for this file? it is empty, it doesn't seem to test anything.

—You are receiving this because you authored the thread.Reply to this email directly or view it on GitHub